src/main/java/com/rolfje/anonimatron/anonymizer/CountryCodeAnonymizer.java

```diff
@@ -36,6 +36,9 @@ public Synonym anonymize(Object from, int size, boolean shortlived) {
 			try {
 				if (size > 2) {
 					country = l.getISO3Country();
+					if(country.length()==0) {
+						continue;
```

Comment:
I think this will always work, but maybe a StringUtils.isEmpty(country) is more robust.

Reply:
Of course you are right. 👍